audbackend/core/backend/base.py

```diff
@@ -9,6 +9,9 @@
 from audbackend.core.errors import BackendError
 
 
+backend_not_opened_error = "'Backend.open()' needs to be run first."
```

Comment:
Maybe be a little more verbose here, e.g.

```suggestion
backend_not_opened_error = "Call 'Backend.open()' to open a connection to the repository first."
```

Reply:
Changed this and the corresponding test accordingly.